build/make/release.mk

```diff
@@ -1,9 +1,11 @@
+##@ Dogu releases
```

Comment:
Since we do way more than releasing dogus (like releasing plain go and k8s binaries) may a rewording is in order.